pkg/cmd/dev/compose.go

```diff
@@ -58,9 +60,41 @@ func (d *dev) compose(cmd *cobra.Command, _ []string) error {
 	if err != nil {
 		return err
 	}
-	cockroachBin := filepath.Join(workspace, "artifacts", "cockroach")
-	compareBin := filepath.Join(workspace, "artifacts", "compare_test")
+	artifactsDir := filepath.Join(workspace, "artifacts")
+	cockroachBin := filepath.Join(artifactsDir, "cockroach")
+	compareBin := filepath.Join(artifactsDir, "compare_test")
 
+	// Copy libgeos files to a temporary directory, since the compose tests expect
+	// libgeos files to be in their own directory.
+	libGeosDir, err := os.MkdirTemp("", "lib")
+	if err != nil {
+		return err
+	}
+	if err := filepath.WalkDir(artifactsDir, func(path string, entry os.DirEntry, err error) error {
```

Comment:
Doesn't seem like there's any reason to do this? Unless I'm missing something, the libraries will be at `artifacts/libgeos.so` and `artifacts/libgeos_c.so`. Just copy those files directly.

Reply:
I wasn't sure if I needed to worry about different file extensions. I guess not.

Reply:
You've hard-coded this to be Linux, so it's not possible for the extensions to differ.